core/services/RestService.js

```diff
@@ -182,7 +182,13 @@ module.exports = {
         ctx.throw(400, "id is required")
       }
 
-      ctx.body = await yaxys.db.update(identity, ctx.params.id, ctx.request.body)
+      try {
```

Comment:
Since we don't do any additional actions in case of error, we don't need a try/catch block here.